core/new-gui/src/app/workspace/component/codearea-custom-template/codearea-custom-template.component.ts

```diff
@@ -20,20 +20,36 @@ import { CoeditorPresenceService } from "../../service/workflow-graph/model/coed
   templateUrl: "./codearea-custom-template.component.html",
   styleUrls: ["./codearea-custom-template.component.scss"],
 })
-export class CodeareaCustomTemplateComponent extends FieldType {
+export class CodeareaCustomTemplateComponent extends FieldType<any> implements AfterViewInit {
   dialogRef: MatDialogRef<CodeEditorDialogComponent> | undefined;
+  disabled: boolean = false;
```

Comment:
I understand we want to disable editing during a workflow execution. However, this particular boolean is describing the CodeareaCustomTemplateComponent, which is never fully "disabled" (i.e., greyed out, not able to click). It shows "View Code" in readonly mode and "Edit Code" in normal mode. The boolean name disabled is confusing here and does not reflect the component's behavior, better change it to be more meaningful.

Reply:
Done.